perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/RegistrarManagerMethod.java

```diff
@@ -479,14 +498,33 @@ public ApplicationFormItem call(ApiCaller ac, Deserializer parms) throws PerunEx
 	 * @param state List<String> List of states: NEW, VERIFIED, APPROVED, REJECTED
 	 * @return List<Application> Found applications
 	 */
+	/*#
+	 * Gets all applications in a given state for a given Group in a given date period.
+	 *
+	 * @param group int Group <code>id</code>
+	 * @param state List<String> List of states: NEW, VERIFIED, APPROVED, REJECTED
+	 * @param dateFrom String Earliest date for applications
+	 * @param dateTo String Latest date for applications
+	 * @return List<Application> Found applications
+	 */
 	getApplicationsForGroup {
 
 		@Override
 		public List<Application> call(ApiCaller ac, Deserializer parms) throws PerunException {
-			if (parms.contains("state")) {
-				return ac.getRegistrarManager().getApplicationsForGroup(ac.getSession(), ac.getGroupById(parms.readInt("group")), parms.readList("state", String.class));
+			if (parms.contains("dateFrom")) {
+				return ac.getRegistrarManager().getApplicationsForGroup(
+						ac.getSession(), 
+						ac.getGroupById(parms.readInt("group")), 
+						parms.contains("state") ? parms.readList("state", String.class) : null,
+						parms.readString("dateFrom"), 
```

Comment:
This is not necessary, but there is also a method for parsing LocalDates. I would suggest using it instead of the String. Same for other similar usages.
```suggestion
						parms.readLocalDate("dateFrom"), 
```